src/components/DateTimePicker/DateTimePicker.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useEffect } from "react";
+import throttle from "lodash.throttle";
+import { useTranslation } from "react-i18next";
+import Picker from "react-datepicker";
+
+import "react-datepicker/dist/react-datepicker.css";
+import CustomInput from './CustomInput';
+
+import en from "date-fns/locale/en-US";
+import ru from "date-fns/locale/ru";
+import hy from "date-fns/locale/hy";
+
+import {
+  DATE_FORMATS,
+  TIME_FORMATS,
+  DATE_TIME_FORMATS
+} from "../../config"
+
+
+const DateTimePicker = props => {
+
+  const LOCALES = { en, ru, hy };
+  const FORMATS = { 
```

Comment:
Is there any way to pass custom format, different than default one?

Reply:
yeah, we can still pass dateFormat prop if we need some special case, but otherwise we can tweak the config file